NewForce-Cohort-4/nutshell-team-odd-eclipse
# Dtb sandbox friends

# Description
I fixed the events and articles issues that we observed during the demo on Tuesday. When we were trying to change the layout and make the sidebar "sticky", the index.html framework changed slightly so the querySelector targets needed to be updated. The main issue was that the modal was moved out of the <main> tag and into <body>. The querySelectors for "Save Article" and "Save Event" were in <main> so they were not functional after the layout changes. 

I also added the friends feature, which allows you to "follow" and "unfollow" friends.

Fixes # Fixed the events and articles save feature. Added friends. 
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Pull down the new branch and double check that everything works. I mean everything. Post a new message. Delete that message. Add a new task, mark it complete. Add a new article. Delete it. Add a new event. Delete it. Follow a new friend. Unfollow someone else.
*You will need to copy API.txt into database.json in order to test the Friends feature.

# Checklist:
